Change pop-up text for publish button on a minor edit document

christophwong opened the pull request:
Previously the pop up text incorrectly says that email will be sent out
for publishing a minor update, it is now changed to match the text used in v1.

Before - publishing a DSU with a minor edit on V2:
![dsu minor v2](https://cloud.githubusercontent.com/assets/3989823/15677349/d5eae1ee-2741-11e6-98b0-39424312371c.png)

After - publishing a DSU with a minor edit on V2:
![after dsu](https://cloud.githubusercontent.com/assets/3989823/15677365/e8f04180-2741-11e6-9736-fe055ef5487e.png)

These examples use DSU, but note that there isn't currently any format-specific differences in the pop up text.

Commit:
Change pop-up text for publish button on a minor edit document

Previously the pop up text incorrectly says that email will be sent out
for publishing a minor update, it is now changed to match the text used
  in v1.

tuzz commented on app/helpers/application_helper.rb:
```diff
@@ -14,4 +14,12 @@ def state(document)
+    else
+      "Publishing will email subscribers to #{current_format.title.pluralize}. Continue?"
+    end
+  end
```
I think this might be better placed in the newly created `PublishWarningPresenter` as it seems to fit with the intent of that presenter. What do you think?

christophwong replied:
We were actually looking to pull that apart and move these frontend logic into helpers rather than presenters. Worth a chat.

tuzz commented on spec/features/publishing_a_cma_case_spec.rb:
```diff
@@ -174,6 +180,8 @@
       expect(page).to have_no_content("You are about to publish a minor edit")
       expect(page).to have_no_content("You are about to publish a major edit")
+
+      expect(page.find_button('Publish')["data-message"]).to eq("Publishing will email subscribers to CMA Cases. Continue?")
```
What do you think about DRYing up these scenarios by having something like:

``` ruby
let(:publish_alert_message) { page.find_button('Publish')["data-message"] }
```

somewhere at the top of the file?

Commit:
Move publish_warning presenter code into helper

After the previous commit we ended up with related code in the
PublishWarningPresenter and a helper. This commit consolidates all of
the code into the helper.
